src/main/java/Duke.java

```diff
@@ -1,12 +1,29 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void echo(String input) {
+        System.out.println(input);
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
         System.out.println("What can I do for you?");
+        Scanner in = new Scanner(System.in);
+        String input;
+        boolean running = true;
+        do {
+            input = in.nextLine();
+            if (input.compareTo("bye") == 0) {
+                running = false;
+            }
+            echo(input);
+        } while (running);
```

Comment:
clear indentation good format

Reply:
clear variable name, indentation is standardised.